.pre-commit-config.yaml

```diff
@@ -16,20 +16,20 @@ repos:
       - id: ruff
         args: ["--fix", "--output-format=full"]
       - id: ruff-format
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.10.0
-    hooks:
-      - id: mypy
-        args: [--ignore-missing-imports]
-        files: ^pymc_marketing/
-        additional_dependencies: [numpy>=1.20, pandas-stubs]
   - repo: https://github.com/nbQA-dev/nbQA
     rev: 1.8.5
     hooks:
       - id: nbqa-ruff
         args: ["--fix", "--output-format=full"]
         files: ^docs/source/notebooks/
         exclude: ^docs/source/notebooks/clv/dev/
+  - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Changing the order so that ruff formatting all happens before mypy (something slow)

Reply:
Can we leave this out of this PR? We want to do this before 0.8

Reply:
Yes. I will change back before this is merged